crates/benda/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "benda"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[lib]
+name = "benda"
+crate-type = ["cdylib"]
+
+[dependencies]
+pyo3 = { version = "0.21.2", features = ["multiple-pymethods"] }
+bend-lang = "0.2.33"
+num-bigint = "0.4.5"
+num-traits = "0.2.19"
+rustpython-parser = "0.3.1"
+indexmap = "2.2.3"
+interner = "0.2.1"
```

Comment:
Please add comments to non-obvious dependencies.

(We are always striving to have fewer dependencies, so it's good to have the non-obvious ones explained)

Reply:
IndexMap is used by Bend to store their ADTs and constructor definitions, so I needed to add it to be able to construct a new IndexMap when parsing a new ADT.